final.js

```diff
@@ -57,18 +67,25 @@ form.addEventListener('submit', (e) => {
     .substr(0, 5);
   // Instantiate Book
   const book = new Book(title, author, id);
+  // Instantiate UI
+  const ui = new UI();
   // Display Book in UI
   myBooks = myBooks.concat(book);
-  addBook(book);
+  ui.addBook(book);
```

Comment:
- [ ] when trying to add a book to the list the user receives the below error, please fix this before requesting another review.
![Screenshot 2022-02-08 221203](https://user-images.githubusercontent.com/39240820/153076695-9999e00a-b0c8-47dc-85fa-8085e9772b29.png)